packages/atdgen/atdgen.2.8.0/opam

```diff
@@ -0,0 +1,105 @@
+description: """
+Atdgen is a command-line program that takes as input type definitions in the ATD
+syntax and produces OCaml code suitable for data serialization and
+deserialization.
+Two data formats are currently supported, these are biniou and JSON.
+Atdgen-biniou and Atdgen-json will refer to Atdgen used in one context or the
+other.
+Atdgen was designed with efficiency and durability in mind. Software authors are
+encouraged to use Atdgen directly and to write tools that may reuse part of
+Atdgen’s source code."""
+maintainer: [
+  "Louis Roché <louis@louisroche.net>"
+  "Martin Jambon <martin@mjambon.com>"
+  "Rudi Grinberg <me@rgrinberg.com>"
+]
+authors: [
+  "Martin Jambon <martin@mjambon.com>"
+  "Rudi Grinberg <rudi.grinberg@gmail.com>"
+  "Martin Jambon <martin@r2c.dev>"
+  "Martin Jambon <github@mjambon.com>"
+  "Ivan Jager <aij+git@mrph.org>"
+  "oleksiy <oleksiy.golovko@ahrefs.com>"
+  "David Sheets <sheets@alum.mit.edu>"
+  "Rudi Grinberg <me@rgrinberg.com>"
+  "Martin Jambon <martin@esper.com>"
+  "Jeff Meister <nanaki@gmail.com>"
+  "Caio Wakamatsu <caio.wakamatsu@ahrefs.com>"
+  "Carmelo Piccione <carmelo.piccione@gmail.com>"
+  "Daniel Weil <danweil68@gmail.com>"
+  "Egor Chemokhonenko <egor.chemohonenko@ahrefs.com>"
+  "Gabriel Scherer <gabriel.scherer@gmail.com>"
+  "Raman Varabets <roman.vorobets@gmail.com>"
+  "tzm <frank@boldsolutions.de>"
+  "Mathieu Baudet <mathieubaudet@fb.com>"
+  "Oleksiy Golovko <alexei.golovko@gmail.com>"
+  "Rauan Mayemir <rauan@mayemir.io>"
+  "Carmelo Piccione <cep1@solvuu.com>"
+  "John Billings <john@monkeynut.org>"
+  "Louis Roché <louis@louisroche.net>"
+  "Brendan Long <self@brendanlong.com>"
+  "Chris Yocum <cyocum@gmail.com>"
+  "Louis Roché (Ahrefs) <louis.roche@ahrefs.com>"
+  "Louis Roché <louis.roche@ahrefs.com>"
+  "Pavel Antoshkin <pavel.antoshkin@ahrefs.com>"
+  "Pierre Boutillier <pierre.boutillier@laposte.net>"
+  "Shon Feder <shon.feder@key.me>"
+  "Anurag Soni <anuragsoni.13@gmail.com>"
+  "Arjun Ravi Narayan <arjunravinarayan@gmail.com>"
+  "Asya-kawai <kawai-toshiki@aintek.xyz>"
+  "Christophe Troestler <christophe.Troestler@umons.ac.be>"
+  "Damien Doligez <ddoligez@janestreet.com>"
+  "Daniel M <dan.mntg@gmail.com>"
+  "Ding Xiang Fei <dingxiangfei2009@protonmail.ch>"
+  "François Pottier <francois.pottier@inria.fr>"
+  "Javier Chavarri <javier.chavarri@gmail.com>"
+  "Kate <kit.ty.kate@disroot.org>"
+  "Louis <louis.roche@ahrefs.com>"
+  "Louis Roché <louis@cryptosense.com>"
+  "Raman Varabets <raman+git@ahrefs.com>"
+  "Stephane Legrand <slegrand45@gmail.com>"
+  "Vincent Bernardoff <vb@luminar.eu.org>"
+  "haoyang <haoyang@esper.co>"
+  "pmundkur <prashanth.mundkur@gmail.com>"
+  "ygrek <ygrek@autistici.org>"
+]
+license: "MIT"
+homepage: "https://github.com/ahrefs/atd"
+bug-reports: "https://github.com/ahrefs/atd/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "atd" {>= "2.7.0"}
+  "atdgen-runtime" {>= "2.0.0"}
```

Comment:
```suggestion
  "atdgen-runtime" {>= "2.1.0"}
```
Packages installing atdgen > 2.1 and atdgen-runtime = 2.0.0 are failing with `Error: Unbound value Atdgen_runtime.Oj_run.missing_field`. We had to update the lower bound to all previous releases to fix this issue: https://github.com/ocaml/opam-repository/pull/21464/commits/d0b4d9527c354de1f25b9c1ea2a9ad00e7daa5df

Reply:
Somehow this cannot be picked here since it would only appears in the lower-bound checks of revdeps

Reply:
great, thank you! I fixed it upstream.